service/database/connector.js

```diff
@@ -42,10 +43,10 @@ class MongoDBConnector {
 
 /** initialize an instance of mongoDB connection and kill process if connection fails */
 const connector = new MongoDBConnector();
-connector.init().catch((e) => {
-  console.error("error connecting to database");
-  process.exit(1);
-});
+// connector.init().catch((e) => {
+//   console.error("error connecting to database");
+//   process.exit(1);
+// });
```

Comment:
I do not think these files are needed anymore as this is being done in caracal.js

Reply:
Yes, you are right! They have been commented out.